### PR TITLE
BC-2528 - enable vuetify everywhere

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/themes/base/vuetify.options.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ import langIconUa from "@assets/icons/lang-icon-ua.vue";
 
 export default {
 	theme: {
+		options: {
```
Comment (uidp) [inline src/themes/base/vuetify.options.js:17]: Why is that moved here from nuxt.config.js?

File: src/layouts/default.vue [hunk 2]
```diff
@@ -1,17 +1,25 @@
 <template>
 	<legacy-logged-in>
-		<base-content-container id="main-content" tag="main" class="content">
-			<Nuxt />
-		</base-content-container>
+		<v-app class="content">
```
Comment (uidp) [inline src/layouts/default.vue:3]: Why are `class` and `id` split up to `<v-app>` and `<v-main>`?

File: src/layouts/default.vue [hunk 3]
```diff
@@ -1,17 +1,25 @@
 <template>
 	<legacy-logged-in>
```
Comment (uidp) [inline src/layouts/default.vue:2]: It looks like `<legacy-logged-in>` is only used here (in this component). Maybe we can refactor that a little differently. Also `<v-app>` is not the root component of the whole layout. Can it be achieved somehow that `<v-app>` wraps the topbar and sidebar as well?

**Review by uidp — COMMENTED** [review]
I would suggest that we look in how we can build a more general default layout that wraps everything in `<v-app>`. Although I'm not sure if that's possible at the current state of things.

File: src/themes/base/vuetify.options.js (hunk 1 above)
Comment (odalys-dataport) [reply src/themes/base/vuetify.options.js:17]: Because it wasn't working in the nuxt.config.js, moving this here made vuetify css vars globally accessible

File: src/layouts/default.vue (hunk 2 above)
Comment (odalys-dataport) [reply src/layouts/default.vue:3]: That's how it was before in `defaultVuetify`. But I can look into it while trying to make `<v-app>` the root component

**Review by uidp — DISMISSED** [review]

**Review by christian-darsow — APPROVED** [review]
